homeassistant/components/aranet/sensor.py

```diff
@@ -81,15 +82,15 @@ class AranetSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.BATTERY,
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
     ),
     "interval": AranetSensorEntityDescription(
         key="update_interval",
         name="Update Interval",
         device_class=SensorDeviceClass.DURATION,
         native_unit_of_measurement=UnitOfTime.SECONDS,
         state_class=SensorStateClass.MEASUREMENT,
-        # The interval setting is not a generally useful entity for most users.
-        entity_registry_enabled_default=False,
```

Comment:
You should keep `entity_registry_enabled_default=False` for this and only add `entity_category=EntityCategory.DIAGNOSTIC`.